runtime/onert/api/nnfw/include/nnfw_experimental.h

```diff
@@ -375,6 +375,28 @@ NNFW_STATUS nnfw_train_get_loss(nnfw_session *session, uint32_t index, float *lo
  */
 NNFW_STATUS nnfw_train_export_circle(nnfw_session *session, const char *path);
 
+/**
+ * @brief Import circle checkpoint
+ * @note  This function should be called on training mode
+ *        This function should be called before {@link nnfw_train}
+ *
+ * @param[in] session The session to export a checkpoint
+ * @param[in] path    The path to export a checkpoint
+ * @return @c NNFW_STATUS_NO_ERROR if successful
+ */
+NNFW_STATUS nnfw_train_import_checkpoint(nnfw_session *session, const char *path);
+
+/**
+ * @brief Export circle checkpoint
+ * @note  This function should be called on training mode
+ *        This function should be called after {@link nnfw_train}
+ *
+ * @param[in] session The session to export a checkpoint
+ * @param[in] path    The path to export a checkpoint
+ * @return @c NNFW_STATUS_NO_ERROR if successful
```

Comment:
(Out of curiosity)
All APIs only describe `NNFW_STATUS_NO_ERROR` for `NNFW_STATUS`. 
Why not explain the other `NNFW_STATUS`s returned from those APIs?

Reply:
@ragmani 

I'm not sure, but maybe it's from nnapi guideline? :)

https://developer.android.com/ndk/reference/group/neural-networks?_gl=1*algvtq*_up*MQ..*_ga*MTI4MTIxNTI0Ni4xNzI0Mjg1MTIy*_ga_6HH9YJMN9M*MTcyNDI4NTEyMi4xLjAuMTcyNDI4NTEyMi4wLjAuMA..#aneuralnetworksmodel_getsupportedoperationsfordevices